data-access-layer/eventmesh/ApiServerClient.js

```diff
@@ -130,6 +199,15 @@ class ApiServerClient {
         }));
   }
 
+  patchResourceStatus(resourceGroup, resourceType, resourceId, statusDelta) {
+    return Promise.try(() => this.init())
```

Comment:
why is this explicity init() done here?

Reply:
To ensure that loadSpec is already loaded.